test/image-blueprints-bootc/layer2-source/group2/rhel95-bootc-source-ai-model-serving.containerfile

```diff
@@ -0,0 +1,87 @@
+# {{- if and (env.Getenv "UNAME_M" "") (eq "x86_64" .Env.UNAME_M) }}
+# Currently, RHOAI is only available for x86_64
+FROM localhost/rhel95-bootc-source:latest
+
+# Build arguments
+ARG USHIFT_RPM_REPO_NAME=microshift-local
+ARG USHIFT_RPM_REPO_PATH=/tmp/$USHIFT_RPM_REPO_NAME
+
+# Copy the MicroShift repository contents
+COPY ./rpm-repos/$USHIFT_RPM_REPO_NAME $USHIFT_RPM_REPO_PATH
+
+# Copy repository configuration
+COPY ./bootc-images/$USHIFT_RPM_REPO_NAME.repo ./bootc-images/microshift-fast-datapath-rhel9.repo ./bootc-images/microshift-rhocp-y.repo \
+    /etc/yum.repos.d/
+
+# Print repository configuration contents.
+# Install MicroShift optional packages and cleanup.
+# qemu-guest-agent is used in offline tests
+# vim-common for xxd utility
+RUN dnf repoinfo --enabled && \
+    dnf install -y \
+        "microshift-ai-model-serving-{{ .Env.SOURCE_VERSION }}" \
+        "microshift-release-info-{{ .Env.SOURCE_VERSION }}" \
+        "microshift-ai-model-serving-release-info-{{ .Env.SOURCE_VERSION }}" \
+        vim-common \
+        qemu-guest-agent && \
+    rm -vf /etc/yum.repos.d/microshift-*.repo && \
+    rm -rvf $USHIFT_RPM_REPO_PATH && \
+    systemctl enable qemu-guest-agent && \
+    dnf clean all
+
+# Override the default qemu-ga service configuration on the guest to allow all RPCs.
+# BLOCK_RPCS is a deny-list of qemu-guest-agent RPCs to block, such as file read/write, process execution, etc. By
+# default, all RPCs are blocked.
+# FREEZE_HOOK_PATHNAME is the dir-path containing hook scripts. This override does not change the default value.
+# These scripts are executed before or after guest image snapshot ops ("freezing" and "thawing" in libvirt speak).
+# No hooks are installed by default.
+RUN printf '\
+\# This is a systemd environment file, not a shell script.\n\
+\# It provides settings for "/lib/systemd/system/qemu-guest-agent.service".\n\
+BLOCK_RPCS=\n\
+FSFREEZE_HOOK_PATHNAME=/etc/qemu-ga/fsfreeze-hook\n' > /etc/sysconfig/qemu-ga
+
+# Build a whitespace separated list of images to be embedded. All images are in single layer.
+#
+# Note:
+# - Retries work around sporadic "cannot set user namespace" podman error.
+RUN --mount=type=secret,id=pullsecret,dst=/run/secrets/pull-secret.json \
+    images="$(jq -r '.images | .[]' /usr/share/microshift/release/release-x86_64.json)" ; \
+    images="${images} $(jq -r '.images | with_entries(select(.key | startswith("kserve") or startswith("ovms-image"))) | .[]' /usr/share/microshift/release/release-ai-model-serving-x86_64.json)" ; \
```

Comment:
Can we add a comment on why we're filtering in specific images?
Also, let's consider splitting the RUN command between the base and ai-model images.